standalone/src/DelphesMain.h

```diff
@@ -74,6 +74,16 @@ int doit(int argc, char* argv[], DelphesInputReader& inputReader) {
       for (auto& [name, coll] : edm4hepConverter.getCollections()) {
         frame.put(std::move(coll), name);
       }
+      // write out LHEF weights once (if available)
+      static bool weightNames_stored = false;
+      if (!weightNames_stored) {
+        const auto& weightNames = inputReader.getWeightNames();
+        if (!weightNames.empty()) {
+          std::cout << "Writing LHEF weight ID vector of strings with size " << weightNames.size() << std::endl;
+          frame.putParameter<std::vector<std::string>>(edm4hep::labels::EventWeightsNames, weightNames);
```

Comment:
```suggestion
          frame.putParameter(edm4hep::labels::EventWeightsNames, weightNames);
```
It's not strictly necessary in this case to explicitly spell out which template function you want to call as the compiler will figure it out from the arguments in any case.